PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -61,3 +66,4 @@ This section is just for reviewers, but it gives a good indication of what revie
 - [ ] Has the version number been revised according to [semantic versioning](https://semver.org)?
 - [ ] Has the changelog been updated accurately?
 - [ ] Is now a good time to merge this PR?
+- [ ] Have the changes made as a result of this pull request been documented in the [Company Wiki](https://wiki.financial-cloud.com)?
```

Comment:
Maybe a slight rewording - something like:

> Do the changes you've made as a result of this pull request require an update to the documentation in the company wiki?

Perhaps another line which says something like:

> Does this pull-request produce any new ideas which should be documented in the wiki?

What do you think? CC @Geenzo @thlgrant

Reply:
@mattbryce93 thoughts?